.github/workflows/testWlsAksWithDependencyCreation.yml

```diff
@@ -19,14 +19,17 @@ on:
         description: 'The VM size for the AKS pool'
         required: true
         default: Standard_D2s_v3
+      location:
+        description: 'The location for the resources'
+        required: true
+        default: eastus
   # sample cURL
   # curl --verbose -X POST https://api.github.com/repos/<github_user>/weblogic-azure/dispatches -H 'Accept: application/vnd.github.everest-preview+json' -H 'Authorization: token <personal_access_token>' --data '{"event_type": "aks-integration-test-with-dependency-creation", "client_payload": {"gitUserNameForArtifactsLocation": "<github_user>", "testBranchNameForArtifactsLocation": "<branch_name>", "isForDemo": "false"}}'
   repository_dispatch:
     types: [aks-integration-test-with-dependency-creation,integration-test-all]
 
 env:
     azureCredentials: ${{ secrets.AZURE_CREDENTIALS }}
-    location: eastus
```

Comment:
So much better to not hard code this!

Reply:
Please file another User Story in the oracle tracker to apply a similar change to the remaining workflows:

```
buildWlsAksArtifact.yml:    location: eastus
setupWlsAksDependency.yml:    location: eastus
testWlsAksWithDependencyCreation.yml:        default: eastus
testWlsAksWithDependencyCreation.yml:                location=eastus # default value
testWlsAksWithoutDependencyCreation.yml:    location: eastus
testWlsVmAdmin.yml:  location: eastus
testWlsVmCluster.yml:  location: eastus
testWlsVmDynamicCluster.yml:  location: eastus
```

Leave it unassigned and put it in the 2025-05 Sprint.